crates/nu-std/tests/test_std.nu

```diff
@@ -8,23 +8,24 @@ def path_add [] {
 
     with-env [$path_name []] {
         def get_path [] { $env | get $path_name }
+        def expand_paths []: list<string> -> list<path> { each { path expand } }
```

Comment:
You don't _need_ `each` here; `path expand` works with lists just fine. Considering this, you don't need this function at all, as you can replace calls to it with just `path expand`.

Reply:
ooh wow, this is very true, good catch :pray:

Reply:
should be good to go in 2eaad1ff6 :wink: